### ISSUE TITLE
White background for printing

### ISSUE THREAD

**Issue by glacialcascade** [edemaine/poketree#3]
It might be nice to have an option to have a white background color (light mode) so that screenshots of trees could be printed without wasting a lot of ink.

**Comment by edemaine** [comment]
Good idea! It was designed for projection, but having a toggle would be nice. This should be easy to do with a toggle and some CSS.

Actually, printing already changes the background to white. Perhaps we just want to tweak the text colors in print mode?
